pkg/morph/client/netmap/config.go

```diff
@@ -172,10 +161,16 @@ func (c *Client) readStringConfig(key string) (string, error) {
 	return v.(string), nil
 }
 
+// reads boolean value by the given key from the NeoFS network configuration
+// stored in the Sidechain. Returns false if key is not presented.
 func (c *Client) readBoolConfig(key string) (bool, error) {
 	v, err := c.config([]byte(key), BoolAssert)
 	if err != nil {
-		return false, err
+		if errors.Is(err, ErrConfigNotFound) {
```

Comment:
why do we need that change?

are we sure that _all_ of our config values gonna be `false` by default?

Reply:
It simplifies readability and allows to reuse the method. Currently all boolean configurations are `false` by default, and IMO it is expected since boolean is false by default almost everywhere. We can always make new config to be `false` default by inverting the meaning.